test/integrations/destinations/pardot/dataDelivery/oauth.ts

```diff
@@ -0,0 +1,53 @@
+import { ProxyV1TestData } from '../../../testTypes';
+import { generateMetadata, generateProxyV1Payload } from '../../../testUtils';
+import { commonRequestParameters, retryStatTags } from './constant';
+
+export const v1OauthScenarios: ProxyV1TestData[] = [
+  {
+    id: 'pardot_v1_oauth_scenario_1',
+    name: 'pardot',
+    description:
```

Comment:
Lets add other scenarios like refresh token, unauthorised access scneario

Reply:
we only have these codes as retryable code [85, 116, 120, 121, 183, 184, 214], rest all will be aborted. have taken them in business not in oauth flow